examples/noise-js/index.js

```diff
@@ -0,0 +1,320 @@
+import { createLightNode } from "js-waku/lib/create_waku";
+import { utils } from "js-waku";
+import { waitForRemotePeer } from "js-waku/lib/wait_for_remote_peer";
+import {
+  Fleet,
+  getPredefinedBootstrapNodes,
+} from "js-waku/lib/predefined_bootstrap_nodes";
+import { PeerDiscoveryStaticPeers } from "js-waku/lib/peer_discovery_static_list";
+import { Protocols } from "js-waku";
+import * as noise from "@waku/noise";
+import protobuf from "protobufjs";
+import QRCode from "qrcode";
+// TODO: Get rid of these
+import hexToArrayBuffer from "hex-to-array-buffer";
+import arrayBufferToHex from "array-buffer-to-hex";
+
+const messagesDiv = document.getElementById("messages");
+const nicknameInput = document.getElementById("nick-input");
+const textInput = document.getElementById("text-input");
+const sendButton = document.getElementById("send-btn");
+const sendingStatusSpan = document.getElementById("sending-status");
+const chatArea = document.getElementById("chat-area");
+const qrCanvas = document.getElementById("qr-canvas");
+const qrUrl = document.getElementById("qr-url");
+const wakuStatusSpan = document.getElementById("waku-status");
+const handshakeStatusSpan = document.getElementById("handshake-status");
+
+function getPairingInfofromUrl() {
+  const urlParts = window.location.href.split("?");
```

Comment:
Can we use a more web idiomatic syntax?

```
const params = new URLSearchParams(window.location.search);

const name = params.get("...");
```

Reply:
For this case it won't make much sense because what we are handling here is not URL parameters but a string that should be parsed in a very specific way.